src/test/java/io/specto/hoverfly/junit/core/HoverflyTest.java

```diff
@@ -32,8 +35,12 @@
     private Hoverfly hoverfly;
     private ObjectMapper mapper = new ObjectMapper();
 
+    @Rule
+    public final RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
```

Comment:
Are you guys started any work on that? If not I am happy to provide this enhancement in the PR as it's currently making my tests super ugly :) I have to deal with it on my own all over the place.